R/select_nse.R

```diff
@@ -236,7 +236,7 @@
 # it is a select helper that we grab from the error message.
 
 .select_symbol <- function(data, x, ignore_case, regex, verbose, ifnotfound) {
-  try_eval <- try(eval(x), silent = TRUE)
+  try_eval <- tryCatch(eval(x), error = function(e) NULL)
```

Comment:
Can you add a comment to say we don't want `try(silent = TRUE)` with a link to this PR?